internal/jimm/cloud.go

```diff
@@ -160,7 +160,7 @@ func (j *JIMM) AddCloudToController(ctx context.Context, user *openfga.User, con
 		return errors.E(op, err)
 	}
 
-	if err := validateCloudRegion(ctx, &j.Database, user, cloud, controllerName); err != nil {
+	if err := validateCloudRegion(ctx, j.Database, user, cloud, controllerName); err != nil {
```

Comment:
It would've been good to keep the mechanical changes to a separate commit/PR if possible.

Reply:
absolutely.. my bad